subcommands/keys/tuf_rotate_offline_key.go

```diff
@@ -108,10 +113,17 @@ func doRotateOfflineKey(cmd *cobra.Command, args []string) {
 		if firstTime && roleName != tufRoleNameRoot {
 			subcommands.DieNotNil(errors.New("First time TUF key rotation must be performed for the root role."))
 		}
+		switch roleName {
```

Comment:
all these switch statements are starting to give off a C code vibe where `#defines` is getting out of hand. i have a feeling there might be a cleaner way to do this so that the reader's eyes are so focused on these details and can instead focus on the broader rotation logic

Reply:
Used the default expiration map: https://github.com/foundriesio/fioctl/pull/184/commits/bc1eeb2de75e2bbf2ee84951fbcae0f109d01cc4 (squash later).

The other switch statements seem sensible to me and really helped to minimize the amount of code.

I know someone would say we need a proper class heirarchy with clearly defined interfaces to avoid the switch and goto statements, because they are religiously impure. But IMHO where a simple switch makes the job we don't need anything else.

Reply:
i don't want you to go crazy on abstractions. you have a feel for the code. If you don't think this will be error prone making future changes where you hit things like "damn i updated the special switch cases for the first 2 swtiches but forgot the minor change for the third".